ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -168,30 +196,37 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
   get fieldsNotEditedAddInstitution() {
     const {institution} = this.state;
     return institution.displayName || institution.userInstructions ||
-      institution.organizationTypeEnum || institution.duaTypeEnum || institution.emailAddresses || institution.emailDomains;
+      institution.organizationTypeEnum || institution.tierRequirements
+        || institution.duaTypeEnum || institution.tierEmailDomains
+        || institution.tierEmailAddresses;
   }
 
   get fieldsNotEditedEditInstitution() {
     const {institution} = this.state;
     return institution.displayName === institutionToEdit.displayName &&
         institution.organizationTypeEnum === institutionToEdit.organizationTypeEnum &&
         institution.duaTypeEnum === institutionToEdit.duaTypeEnum &&
-        institution.emailAddresses === institutionToEdit.emailAddresses &&
-        institution.emailDomains === institutionToEdit.emailDomains &&
+        institution.tierEmailAddresses === institutionToEdit.tierEmailAddresses &&
+        institution.tierEmailDomains === institutionToEdit.tierEmailDomains &&
+        institution.tierRequirements === institutionToEdit.tierRequirements &&
         institution.userInstructions === institutionToEdit.userInstructions &&
         institution.organizationTypeOtherText === institutionToEdit.organizationTypeOtherText;
   }
 
-
   validateRequiredFields() {
```

Comment:
This is also a strange confusing function.  When all fields are valid, we want this to return FALSE.  Please retitle it so that it makes sense.  Something like "hasInvalidFields()"

Please also rename the variables in this function so they also make sense.